controllers/vspherecluster_reconciler.go

```diff
@@ -51,12 +51,6 @@ import (
 	infrautilv1 "sigs.k8s.io/cluster-api-provider-vsphere/pkg/util"
 )
 
-// legacyIdentityFinalizer is deprecated and should be used only while upgrading the cluster
-// from v1alpha3(v.0.7).
-//
-// Deprecated: legacyIdentityFinalizer will be removed in a future release.
-const legacyIdentityFinalizer string = "identity/infrastructure.cluster.x-k8s.io"
```

Comment:
Can you comment on https://github.com/kubernetes-sigs/cluster-api-provider-vsphere/issues/2394 that this has been removed? 

I think we can go ahead with this as it's related to v1alpha3 -> v1beta1 changes rather than anything to do with capw transition AFAICT.

Reply:
Done